app/services/fees/subscription_service.rb

```diff
@@ -137,7 +137,11 @@ def terminated_amount
       # NOTE: number of days between beginning of the period and the termination date
       number_of_day_to_bill = (to_datetime - from_datetime).fdiv(1.day).ceil
 
-      number_of_day_to_bill * single_day_price(subscription)
+      number_of_day_to_bill *
+      single_day_price(
```

Comment:
Layout/MultilineOperationIndentation: Use 2 (not 0) spaces for indenting an expression spanning multiple lines.